splice_machine/src/main/java/com/splicemachine/derby/stream/function/MergeAllAggregatesFunction.java

```diff
@@ -43,7 +43,6 @@ public ExecRow call(ExecRow locatedRow1, ExecRow locatedRow2) throws Exception {
             aggregates = op.aggregates;
             initialized = true;
         }
-        operationContext.recordRead();
```

Comment:
Why is this line removed?

Reply:
It's an unrelated change, we were overcounting rows produced on GroupedAggregattes. I'd rather leave it in since it's a very simple change.

Reply:
sure, thanks!